model/result.go

```diff
@@ -45,4 +45,10 @@ type Result struct {
 	//
 	// Indexed the same as Challenge.Tasks.
 	FinalSubmissions []*datastore.Key
+
+	// The time when the coder started the challenge
+	Started,
+
+	// The time when the coder finished the challenge
```

Comment:
Missing punctuation. Also, you could spell Challenge with a capital C, as you are referring to the property.